troubleshooting/kubernetes/Troubleshoot_Kubernetes_Pods_Not_Starting.md

```diff
@@ -0,0 +1,37 @@
+# Troubleshoot Kubernetes Pods Not Starting
+
+Use this procedure to check if Kubernetes pods get scheduled on an NCN, but don't eventually reach the `Running` state.
+
+## Prerequisites
+
+The `kubectl get pod` command returns pods that seem to be stuck in the `Init` or `ContainerCreating` state.
+
+## Identify the node in question
+
+1. Run the `kubectl get pod -o wide` command to identify the node where the pod isn't starting.
+
+    ```bash
+    ncn-w001# kubectl get pod -A -o wide | grep 'Init\|ContainerCreating'
```

Comment:
alternatively you could use egrep 'Init|ContainerCreating'

Reply:
sounds good -- will change